lib/phoenix_test/playwright.ex

```diff
@@ -283,6 +258,63 @@ defmodule PhoenixTest.Playwright do
     session
   end
 
+  @doc """
+  Focuses the matching element and simulates user typing.
+
+  In most cases, you should use `fill_in/4` instead.
+
+  ## Options
+  - `:delay` (integer): Time to wait between key presses in milliseconds. Defaults to 0.
+
+  ## Examples
+      > PhoenixTest.Playwright.type(session, "#id", "some text")
+  """
+  def type(session, selector, text, opts \\ []) when is_binary(text) do
```

Comment:
Usage of `selector` is not currently standardized.

In most cases (implementing standard `PhoenixTest` protocol functions) it is handled as a CSS selector.

In Playwright-specific functions, it is treated as Playwright selector. Which in most cases gracefully handles CSS without the `css=` prefix. While still allowing the caller to optionally use advanced selectors such as `Selector.label`.

Seems to work for now. Should be better documented. Might need more explicit handling in the future (e.g.`"#css-as-default"` vs `{:playwright, "internal:label=..."}`)